File: web_site/src/entities/GetMessageDto.ts
export type GetMessageDto =
{
  id_Message: number,
  message_content: string,
  date_of_request: Date,
  idUserIssuer: number,
  id_UserReceiver: number,
  userReceiverPseudo: string,
  userIssuerPseudo: string,
}

File: src/Domain/Dtos/GetDtos/GetMessageDto.cs
namespace Domain.Dtos.GetDtos
{
    public class GetMessageDto
    {
        public int Id_Message { get; set; }
        public string message_content { get; set; }
        public DateTime Date_of_request { get; set; }
        public int IdUserIssuer { get; set; }
        public int Id_UserReceiver { get; set; }
        public string UserReceiverPseudo { get; set; }
        public string UserIssuerPseudo { get; set; }
    }
}